### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?184687
@millerdev

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 5 errors:

* touchforms/formplayer/views.py, line 134 - E302 expected 2 blank lines, found 1
* touchforms/formplayer/views.py, line 162 - E302 expected 2 blank lines, found 1
* touchforms/formplayer/views.py, line 166 - E302 expected 2 blank lines, found 1
* touchforms/formplayer/views.py, line 176 - E302 expected 2 blank lines, found 1
* touchforms/formplayer/views.py, line 203 - E302 expected 2 blank lines, found 1

File: touchforms/formplayer/views.py [hunk 1]
```diff
@@ -97,22 +112,26 @@ def enter_form(request, **kwargs):
         session_data['preloaders'] = preload_data
 
     if not xform:
-        xform = get_object_or_404(XForm, id=xform_id)
+        try:
+            xform = get_object_or_404(XForm, id=xform_id)
+        except ValueError:
```
Comment (millerdev) [inline touchforms/formplayer/views.py]: That's a surprising exception to come out of `get_object_or_404`. Seems like it maybe should be fixed deeper inside that function?

Comment (snopoke) [inline touchforms/formplayer/views.py]: That's a django function

Comment (millerdev) [inline touchforms/formplayer/views.py]: heh, ok

File: touchforms/formplayer/templates/touchforms/touchscreen.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/deprecate/formnav.js?version={{app_version}}"></script>
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/vendor/dates.js?version={{app_version}}"></script>
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/deprecate/entrycontrols.js?version={{app_version}}"></script>
-    <script language="javascript" type="text/javascript;version=1.7" src="{{STATIC_URL}}formplayer/script/decprecate/workflow.js?version={{app_version}}"></script>
+    <script language="javascript" type="text/javascript;version=1.7" src="{{STATIC_URL}}formplayer/script/deprecate/workflow.js?version={{app_version}}"></script>
```
Comment (benrudolph) [inline touchforms/formplayer/templates/touchforms/touchscreen.html]: how did you discover this? do people still use this page?

Comment (snopoke) [inline touchforms/formplayer/templates/touchforms/touchscreen.html]: No I don't think so. I really wanted to just delete it but wanted to chat to you first. If you access `/webforms/` locally you can get to this page.

**Pull request merged**